exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -1,54 +1,71 @@
-from exasol_sphinx_github_pages_generator.cli import Parser, Console
+import click
+from exasol_sphinx_github_pages_generator.cli import Console
 from exasol_sphinx_github_pages_generator.deployer import GithubPagesDeployer
 from tempfile import TemporaryDirectory
 from subprocess import run
 import os
-import sys
 from inspect import cleandoc
 
 
-def deploy_github_pages(argv):
+@click.command()
```

Comment:
Due to the fact that this only contains the cli interface I suggest you merge the contents of this file with the `cli.py` module